tests/ui/diagnostic-width/E0271.rs

```diff
@@ -15,7 +17,7 @@ impl<T> Future for Option<T> {
 struct Foo;
 
 fn foo() -> Box<dyn Future<Error=Foo>> {
-    Box::new( //~ ERROR E0271
+    Box::new( //[ascii]~ ERROR E0271
```

Comment:
Oh, so `compiletest` can't handle `human-unicode` yet?

Reply:
Kind of: when modifying error-format we're overriding the implicit `--error-format=json` that compiletest uses. This change predates me handling the forwarding of the underlying rendered field correctly, so we can change this test to be `--error-format=json --json=diagnostic-unicode` and it should Just Work️™️